guide/content/stylesheets/application.scss

```
@@ -4,7 +4,6 @@ $govuk-assets-path: "/assets/";
 $govuk-font-family: system-ui, sans-serif;
 $govuk-brand-colour: #2288aa;
 $govuk-link-colour: #006688;
-$govuk-hover-colour: #004466;
 $govuk-visited-colour: #333366;
```

Oops, these 2 colour variables should have link in their name:

```suggestion
$govuk-link-hover-colour: #004466;
$govuk-link-visited-colour: #333366;
```